depthai_ros_driver/src/pipelines/stereo_with_mobilened_ssd_pipeline.cpp

```diff
@@ -0,0 +1,133 @@
+#include <ros/console.h>
```

Comment:
I deleted this include from other pipeline cpps based on the comments in https://github.com/rapyuta-robotics/depthai_ros/pull/46

Reply:
isn't it needed for ROS_INFO or other loggin?